staff/pedro-garriganogues/gamma-template/sass.sass

```diff
@@ -0,0 +1,235 @@
+.burger
```

Comment:
usually the name of the styles files is styles.sass

Reply:
Going to do that from now on!